clouditor-engine-core/src/main/java/io/clouditor/assurance/Certification.java

```diff
@@ -133,4 +122,52 @@ public String getWebsite() {
   public void setWebsite(String website) {
     this.website = website;
   }
+
+  @Override
+  public String toString() {
+    return "Certification{"
```

Comment:
Isn't there a ToStringBuilder somewhere already in the classpath? Seems unmaintanable to have it hardcoded here